client/flutter/lib/components/page_button.dart

```diff
@@ -61,14 +61,34 @@ class _PageButtonState extends State<PageButton> {
 
   @override
   Widget build(BuildContext context) {
-    return FlatButton(
+    return TextButton(
       onPressed: widget.onPressed != null ? _onPressed : null,
-      disabledColor: Constants.neutralTextLightColor.withOpacity(0.4),
-      disabledTextColor: Colors.white,
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(widget.borderRadius),
-      ),
-      color: widget.backgroundColor,
+      style: ButtonStyle(
+          foregroundColor: MaterialStateProperty.resolveWith(
+            (states) {
+              if (states.contains(MaterialState.disabled)) {
+                return Colors.white;
+              } else {
+                return null;
+              }
+            },
+          ),
+          backgroundColor: MaterialStateProperty.resolveWith(
+            (states) {
+              if (states.contains(MaterialState.disabled)) {
+                return Constants.neutralTextLightColor.withOpacity(0.4);
+              } else {
+                return widget.backgroundColor;
```

Comment:
Just a guess whether `widget.backgroundColor` is used as the default or does this need to be explicitly set?

Reply:
`widget.backgroundColor` is not the default. It is passed into the `pageButton` class and then put into the button.